activestorage/test/models/attached/many_test.rb

```diff
@@ -726,7 +726,7 @@ class ActiveStorage::ManyAttachedTest < ActiveSupport::TestCase
 
   test "clearing change on reload" do
     @user.highlights = [ create_blob(filename: "funky.jpg"), create_blob(filename: "town.jpg") ]
-    assert @user.highlights.attached?
+    assert_predicate @user.highlights, :attached?
 
     @user.reload
     assert_not @user.highlights.attached?
```

Comment:
I think we should extend the rubocop rule to also account for `assert_not` -> `assert_not_predicate` usages